test/util/authenticate-user.js

```diff
@@ -1,7 +1,8 @@
 // Allow main functionality to stay at top of file:
 /* eslint-disable no-use-before-define */
 
-const makeFetchCookie = require('fetch-cookie');
+// fetch-cookie v3+ is an ES module, so we need to destructure the default export when using require()
```

Comment:
This comment may be redundant if we'll be doing this loads in the future - I suspect this is notation which we'll have to get used to.